deploy/contents/sync/scrap/scrap.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+#This script connects and gather Kubernetes ConfigMap, Cloud-Pipeline preferences, List of the installed cp services and installed pipectl version
+# from provided server address and (optionally) IP of the Kubernetes Node.
+
+#Example of usage ./scrap.sh  -k <ssh_key name>  -u <user who connect to server> -s <server adress name> -t <api_token> -o <output directory name> 
+
+source get_configmap.sh
+source get_pref.sh
+source get_services.sh
+source get_version.sh
+
+
+while [[ $# -gt 0 ]]
+    do
+    key="$1"
+
+    case $key in
+        -k)
+        ssh_key="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -u)
+        user="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -s)
+        server_address="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -i)
+        node_ip="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -t)
+        token="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -n)
+        namespace="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -c)
+        configmap_name="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -o)
+        output_dir="$2"
+        shift # past argument
+        shift # past value
+        ;;
+esac        
+done
+
+if [ -z "$ssh_key" ]; then
+    echo "Please provide the ssh key"
+    exit 1
+fi
+
+if [ -z "$server_address" ]; then
```

Comment:
in many scripts we have our well defined `API_URL` and `API_TOKEN` env vars, lets use it here as a second thing to check:
 - if it present - use it as a value
 - if not, fail as we do now